src/pages/About/About.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import Icon from "./Icon";
-import "./style/Study.css";
+import Icon from "../../components/Icon/Icon";
+import "../../style/Study.css";
```

Comment:
As `Study.css` is used in a number of places I have left it in the top level `style` folder. Maybe the classes could be broken up to the specific components that use them, but don't think that's really necessary.